src/layout/landingPage/styles/Body.scss

```diff
@@ -0,0 +1,24 @@
+.ins-c-landing-body {
+  height: 40vh;
+  margin-top: 48px;
+  margin-right: 24px;
+  margin-bottom: 32px;
+}
+
+.ins-c-landing-grouping {
+  margin-right: 24px;
+  margin-left: 24px;
+
+  .ins-c-landing-separator {
+    margin-bottom: 32px;
+  }
+
+  .ins-c-landing-grouping__title {
+    margin-bottom: var(--ins-global-spacer--xl);
+    
+  }
+
+  .ins-c-landing-spacer-grouping {
+    height:32px;
+  }
```

Comment:
Whenever possible, try to use pf variables here instead of redefining them as `ins` or using `px` values